charts/azuremonitor-containers/templates/ama-logs-daemonset-windows.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if or (and ( not (contains "microsoft.kubernetes/connectedclusters" (.Values.Azure.Cluster.ResourceId | lower))) ( not (contains "microsoft.hybridcontainerservice/provisionedclusters" (.Values.Azure.Cluster.ResourceId | lower)))) ( not (.Values.amalogs.useAADAuth)) }}
```

Comment:
below condition should be good enough to cover any cluster with useAADAuth=false, right?

{{- if ( not (.Values.amalogs.useAADAuth)) }}

Reply:
Yeah that is also true....Let me change...I thought lets just change for extension but we can block for others as well